ios-target/ViewModel/Home/HomeViewModel.swift

```diff
@@ -100,11 +101,16 @@ class HomeViewModel {
   
   func loadTargetPoints() {
     state = .loading
-    TargetAPI.getTargets({ [weak self] targets in
-      self?.targets = targets
-      self?.addAnnotations(targets: targets)
-      self?.state = .idle
-      }, failure: { [weak self] error in
+    TopicAPI.getTopics({ [weak self] topics in
+      self?.topics = topics
+      TargetAPI.getTargets({ [weak self] targets in
```

Comment:
I think you can do both calls at the same time with a dispatch group since there are no dependencies between them.